packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/main.tsp

```diff
@@ -38,3 +38,32 @@ union ProvisioningState {
   Deleting: "Deleting",
   Accepted: "Accepted",
 }
+
+@armResourceOperations
+interface ListingAvailableOperations {
```

Comment:
Not sure if we need an extra interface for this..

Reply:
OK, seems for cadl-ranch, we need that. Maybe just `AvailableOperations`, since `list` is in this interface.

Reply:
Changed `ListingAvailableOperations` to `AvailableOperations`.